### DISCUSSION TITLE
Server API

### DISCUSSION THREAD

**Discussion by juanitoddd** [music-assistant/support#2539] [Q&A] [answered]
Hello dear Music Assistant friends

Forgive me if this has been answered somewhere else (could not find docs) but,

Is there some Rest/Websockets API one could call on the server ?  in order to perform basic tasks 

I have some players (squeeze) that I would like to sync/un-sync programatically using python.

Thanks & Thank you for your efforts towards this cool project

**Comment by OzGav** [comment, marked as answer]
No but joining players can be done via the HA service calls

**Reply by juanitoddd** [reply]
@OzGav Thank you very much ! Just tried out and worked flawlessly

**Comment by OzGav** [comment]
There is now API documentation for the 2.7.0 beta and later versions of Music Assistant. The API documentation is automatically generated and available at http://your_ma_server_ip:8095/api-docs
